CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+- Issue when running `sedge` without sudo.
```

Comment:
We also have an issue with running `sedge deps` with or without sudo, therefore I would like to update this log with something more related to this fix.

Reply:
Fixed in 1741c7fb72189f23933f99ef0e8c1657ee643598